### PR TITLE
Installed RPM Into the Five Authorization Docker Images

### PR DESCRIPTION
**Pull request by jackieung-dell**
# Description
Updated the shell script to install RPM into the 5 Authorization Docker Images

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/1221|

# Checklist:

- [*] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [* ] I have verified that new and existing unit tests pass locally with my changes
- [* ] I have not allowed coverage numbers to degenerate
- [* ] I have maintained at least 90% code coverage
- [* ] I have commented my code, particularly in hard-to-understand areas
- [* ] I have made corresponding changes to the documentation
- [* ] I have added tests that prove my fix is effective or that my feature works
- [* ] Backward compatibility is not broken

# How Has This Been Tested?
Ran the Makefile onto another Dev Machine and verified in each of the 5 Authorization Docker Images that RPM was installed.

### CODE REVIEW

**Review by donatwork — APPROVED** [review]

**Review by Sahiba-Gupta — APPROVED** [review]

**Review by atye — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -139,3 +138,9 @@ download-csm-common:
 .PHONY: lint
 lint: 
 	golangci-lint run --fix
+
+.PHONY: build-base-image
+build-base-image: download-csm-common
+	$(eval include csm-common.mk)
+	sh ./scripts/build_ubi_micro.sh $(DEFAULT_BASEIMAGE)
+	$(eval BASEIMAGE=authorization-ubimicro:latest)
```
Comment (shaynafinocchiaro) [inline Makefile:146]: add new line

File: scripts/build_ubi_micro.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+microcontainer=$(buildah from $1)
+micromount=$(buildah mount $microcontainer)
+dnf install --installroot $micromount --releasever=8 --nodocs --setopt install_weak_deps=false --setopt=reposdir=/etc/yum.repos.d/ rpm -y
+dnf clean all --installroot $micromount
+buildah umount $microcontainer
+buildah commit $microcontainer authorization-ubimicro
```
Comment (shaynafinocchiaro) [inline scripts/build_ubi_micro.sh:7]: add new line

File: Makefile (hunk 1 above)
Comment (jackieung-dell) [reply Makefile]: Added a new line

File: scripts/build_ubi_micro.sh (hunk 2 above)
Comment (jackieung-dell) [reply scripts/build_ubi_micro.sh]: Added a new line

**Review by shaynafinocchiaro — APPROVED** [review]

**Review by atye — APPROVED** [review]

**Comment by atye** [comment]
e2e pass

**Pull request merged**